README.md

```diff
@@ -98,13 +98,10 @@ See our [Configuration Guide](CONFIG.md) for various configuration options, incl
 Note that the validator also has a [batch processing mode](gtfs-realtime-validator-lib/README.md#batch-processing) - see the [**gtfs-realtime-validator-lib** README](gtfs-realtime-validator-lib/README.md).
  
 ## Docker
- 
-Pull the latest image from [Docker Hub](https://hub.docker.com/r/cutrusf/gtfs-realtime-validator):
-
-`docker pull cutrusf/gtfs-realtime-validator`
- 
+### Setup
+1. Download and install [Docker](https://docs.docker.com/get-started/)
+2. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-realtime-validator). For example, `docker pull ghcr.io/mobilitydata/gtfs-realtime-validator` for the latest snapshot version of the validator.
```

Comment:
To make this easier to read I'd split the lines to put the commands on their own lines, like we have it currently.

Reply:
Done in https://github.com/MobilityData/gtfs-realtime-validator/pull/86/commits/48fce5ba552e9336c1b339ae2194168b43bde081